instance_dedicated_gpu-alpha.tf

```diff
@@ -3,7 +3,7 @@ variable "gpu-node-alpha-dns" {
 }
 
 data "openstack_images_image_v2" "gpu-node-alpha-image" {
-  name = "vggp-gpu-v60-j310-1fad751e0150-main"
+  name = "vggp-v60-gpu-j322-692e75a7c101-main"
```

Comment:
why has the name shifted from gpu-v60 to v60-gpu?

Reply:
I wondered the same. I copied the image from our [VGCN infrastructure](https://github.com/usegalaxy-eu/vgcn-infrastructure/blob/main/resources.yaml#L7C12-L7C47). I will later look when was this image created and from next versions let's stick to one naming scheme.